WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -354,104 +355,142 @@ def getAtticJobParameters( self, jobID, paramList = None, rescheduleCounter = -1
     """ Get all Job Attributes for a given jobID.
         Return a dictionary with all Job Attributes,
         return an empty dictionary if matching job found
     """
 
-    ret = self._escapeString( jobID )
+    ret = self._escapeString(jobID)
     if not ret['OK']:
       return ret
     jobID = ret['Value']
 
     if attrList:
       attrNameList = []
       for x in attrList:
-        ret = self._escapeString( x )
+        ret = self._escapeString(x)
         if not ret['OK']:
           return ret
         x = "`" + ret['Value'][1:-1] + "`"
-        attrNameList.append( x )
-      attrNames = ','.join( attrNameList )
+        attrNameList.append(x)
+      attrNames = ','.join(attrNameList)
     else:
       attrNameList = []
       for x in self.jobAttributeNames:
-        ret = self._escapeString( x )
+        ret = self._escapeString(x)
         if not ret['OK']:
           return ret
         x = "`" + ret['Value'][1:-1] + "`"
-        attrNameList.append( x )
-      attrNames = ','.join( attrNameList )
-    self.log.debug( 'JobDB.getAllJobAttributes: Getting Attributes for job = %s.' % jobID )
+        attrNameList.append(x)
+      attrNames = ','.join(attrNameList)
+    self.log.debug('JobDB.getAllJobAttributes: Getting Attributes for job = %s.' % jobID)
 
-    cmd = 'SELECT %s FROM Jobs WHERE JobID=%s' % ( attrNames, jobID )
-    res = self._query( cmd )
+    cmd = 'SELECT %s FROM Jobs WHERE JobID=%s' % (attrNames, jobID)
+    res = self._query(cmd)
     if not res['OK']:
       return res
 
     if not res['Value']:
-      return S_OK ( {} )
+      return S_OK({})
 
     values = res['Value'][0]
 
     attributes = {}
     if attrList:
       for i in xrange(len(attrList)):
-        attributes[attrList[i]] = str( values[i] )
+        attributes[attrList[i]] = str(values[i])
     else:
       for i in xrange(len(self.jobAttributeNames)):
-        attributes[self.jobAttributeNames[i]] = str( values[i] )
+        attributes[self.jobAttributeNames[i]] = str(values[i])
 
-    return S_OK( attributes )
+    return S_OK(attributes)
 
 #############################################################################
-  def getJobAttribute( self, jobID, attribute ):
+  def getJobAttribute(self, jobID, attribute):
     """ Get the given attribute of a job specified by its jobID
     """
 
-    result = self.getJobAttributes( jobID, [attribute] )
+    result = self.getJobAttributes(jobID, [attribute])
     if result['OK']:
       value = result['Value'][attribute]
-      return S_OK( value )
+      return S_OK(value)
 
     return result
 
 #############################################################################
-  def getJobParameter( self, jobID, parameter ):
+  def getJobStatus(self, jobID):
+    """ Get all Job Status values for a given jobID.
+
+    :param self: self reference
+    :param int jobID: Job ID
+
+    :return : dict with all Job Status values/empty dict if matching job not found
```

Comment:
I think the space after `:return :` will break the build of the doc

Reply:
No, it works.